dbschema/src/main/java/org/folio/rest/persist/ddlgen/Index.java

```diff
@@ -55,5 +57,36 @@ public String getArraySubfield() {
   public void setArraySubfield(String modifiersSubfield) {
     this.arraySubfield = modifiersSubfield;
   }
+  public String getMultiFieldNames() {
+    return multiFieldNames;
+  }
+  public void setMultiFieldNames(String queryIndexName) {
+    this.multiFieldNames = queryIndexName;
+  }
+  public String getSqlExpression() {
+    return sqlExpression;
+  }
+  public void setSqlExpression(String sqlExpression) {
+    this.sqlExpression = sqlExpression;
+  }
+  public String createCompoundIndex(String tableLoc) {
```

Comment:
Can you rename it to getFinalSqlExpression(String) table? fieldPath is not compound.